src/Application.vala

```diff
@@ -286,6 +286,11 @@ public class AppCenter.App : Gtk.Application {
                     if (main_window != null) {
                         var win = main_window.get_window ();
                         if (win != null && (win.get_state () & Gdk.WindowState.FOCUSED) != 0) {
+                            main_window.toast.title = _("%s has been successfully installed").printf (package.get_name ());
```

Comment:
I know this mirrors other copy from the notification, but "successfully" is redundant. It either installed, or it didn't :smile: 

```suggestion
                            main_window.toast.title = _("%s has been installed").printf (package.get_name ());
```